packages/pyright-internal/src/analyzer/program.ts

```diff
@@ -2032,6 +2032,10 @@ export class Program {
             }
 
             // We might not have the file info if binding failed for whatever reasons.
+            if (!AnalyzerNodeInfo.hasAnalyzerInfo(parseResults.parseTree)) {
+                continue;
+            }
+            
             const fileInfo = AnalyzerNodeInfo.getFileInfo(parseResults.parseTree);
             if (fileInfo && fileInfo.accessedSymbolSet) {
```

Comment:
and `getFileInfo` can't return `undefined`, so you shouldn't need to check `fileInfo` in `if statement`?